content/en/docs/application-store/built-in-apps/tomcat-app.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Deploy Tomcat"
+keywords: 'demploy, tomcat'
+description: 'Deploy Tomcat'
+
+linkTitle: "Deploy Tomcat"
+weight: 2110
+---
+
+This tutorial walks you through an example of how to demploy tomcat.
```

Comment:
You would better to add a simple introduction to Tomcat and add its official website link to Tomcat at the beginning.